specs-go/config.go

```diff
@@ -615,6 +615,25 @@ type Arch string
 // LinuxSeccompFlag is a flag to pass to seccomp(2).
 type LinuxSeccompFlag string
 
+const (
+	// LinuxSeccompFlagLog is the filter to return actions except
+	// SECCOMP_RET_ALLOW should be logged. An administrator may override this
+	// filter flag by preventing specific actions from being logged via the
+	// /proc/sys/kernel/seccomp/actions_logged file. (since Linux 4.14)
+	LinuxSeccompFlagLog LinuxSeccompFlag = "SECCOMP_FILTER_FLAG_LOG"
+
+	// LinuxSeccompFlagTsync is the filter to synchronize all other threads of
+	// the calling process to the same seccomp filter tree. A "filter tree" is
+	// the ordered list of filters attached to a thread. (Attaching identical
+	// filters in separate seccomp() calls results in different filters from
+	// this perspective.)
+	LinuxSeccompFlagTsync LinuxSeccompFlag = "SECCOMP_FILTER_FLAG_TSYNC"
+
+	// LinuxSeccompFlagSpecALlow can be used to disable Speculative Store
+	// Bypass mitigation. (since Linux 4.17)
+	LinuxSeccompFlagSpecALlow LinuxSeccompFlag = "SECCOMP_FILTER_FLAG_SPEC_ALLOW"
```

Comment:
```suggestion
	// LinuxSeccompFlagSpecAllow can be used to disable Speculative Store
	// Bypass mitigation. (since Linux 4.17)
	LinuxSeccompFlagSpecAllow LinuxSeccompFlag = "SECCOMP_FILTER_FLAG_SPEC_ALLOW"
```